cli/bench/deno_tcp.ts

```
@@ -13,15 +13,11 @@ async function handle(conn: Deno.Conn): Promise<void> {
   const buffer = new Uint8Array(1024);
   try {
     while (true) {
-      const r = await conn.read(buffer);
-      if (r === null) {
-        break;
-      }
+      await conn.read(buffer);
       await conn.write(response);
     }
-  } finally {
-    conn.close();
-  }
+  } catch { /**/ }
```

Maybe print put error instead of silently swallowing?